tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/c_api/AbstractTF_Tensor.java

```diff
@@ -32,14 +32,7 @@
     /** TensorFlow crashes if we don't pass it a deallocator, so... */
     protected static Deallocator_Pointer_long_Pointer dummyDeallocator = new Deallocator_Pointer_long_Pointer() {
         @Override public void call(Pointer data, long len, Pointer arg) { }
-    };
-
-    static {
-        PointerScope s = PointerScope.getInnerScope();
-        if (s != null) {
-            s.detach(dummyDeallocator);
-        }
-    }
+    }.retainReference();
```

Comment:
This change doesn't look related to the rest of the PR?

Reply:
It's needed for JavaCPP 1.5.2, which changed a bit how `PointerScope` works to accommodate for reference counting. It doesn't affect anything since it's not being used right now, and it might never be either, if that gets fixed upstream and this hack is no longer needed.